config/settings/development.yml

```diff
@@ -10,3 +10,5 @@ omniauth:
     enable: false
 oai_pmh:
   admin_mail: admin@example.org
+open_ai:
+  access_token: ~ #ENTER ACCESS TOKEN HERE
```

Comment:
Usually, we place at least a space between the comment and the hashtag. Maybe, we could also add a reference to https://platform.openai.com/api-keys, since users can generate an API key there.

This comment applies to all files in this directory.